src/lib/api.ts

```diff
@@ -7,12 +7,22 @@ class Api {
   private _rootTypes: Type[] = [];
   private _rootNamespaces: Namespace[] = [];
 
+  private _structure: string = 'flat';
```

Comment:
This isn't the right place to put this, as it's not concerned with the builder/parsed data which is what the `Api` class entails (it's not named the best sorry). So instead we should be passing this through to the template renderer/exporter directly instead of this third-party approach.